app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if len(command.split()) == 3:
+        command_cp, file_name, file_to_copy = command.split()
```

Comment:
don't split command twice command.split()
create a separated variable and call it on demand